aiida_vasp/parsers/vasp.py

```diff
@@ -285,8 +325,9 @@ def _check_and_validate_settings(self):
                         missing_files.append(missing_file)
 
                 missing_files = ", ".join(missing_files)
-                self.logger.warning('{0} has been requested, however the following files'.format(quantity) +
-                                    ' required for parsing have not been retrieved: {0}.'.format(missing_files))
+                self.logger.warning('{0} has been requested, however the following files'
```

Comment:
Although being correct code, could you make it more readable, something like
```
message = '{quantity} has been requested, ...'.
message += ' required ... {missing_files}'
message = message.format(quantity=quantity, missing_files=missing_files)
```

Reply:
Agreed, but did it a bit differently.